src/modules/dashboard/checks.ts

```diff
@@ -28,89 +29,93 @@ export const didClaimPayout = (
   userAddress: Address,
 ) => taskUser && taskUser.didClaimPayout && taskUser.address === userAddress;
 
-export const isManager = ({ managerAddress }: TaskType, userAddress: Address) =>
+export const isManager = ({ managerAddress }: AnyTask, userAddress: Address) =>
   managerAddress === userAddress;
```

Comment:
Not really sure what to do for this check... is manager is different from creator?